### ISSUE TITLE
Time formatting

### ISSUE THREAD

**Issue by JackAtOmenApps** [jckw/django-bootstrap-daterangepicker#3]
This seems too small for a pull request, but I'll supply one if you'd like.

widgets.py, lines 23 and 24 should be changed from

```
    '%H': 'hh',
    '%I': 'h',
```

to 

```
    '%H': 'HH',
    '%I': 'hh',
```

Both **%H** and **HH** represent zero-padded 24-hour hours, so one should be equivalent to the other. **h** represents non-padded 12-hour hours, and is probably not useful with this widget. **%I** and **hh** are both zero-padded 12-hour hours, and should be considered equivalent as well.

If "picker_options={'timePicker24Hour': True,}" is passed to the DateTimepicker, and %H is used in the format string, the widget popup will display 24-hour hours, but will send 12-hour hours to the CharField in the form. For instance, the user can actually select 1930 or 2300 as valid times in the popup, but 0730 and 1100 will result in the text of the CharField.

Additionally, the default example on the Readme, which uses 12-hour hours, should probably use **'%I'**, rather than **'%H'**

References:
http://strftime.org
https://devhints.io/moment

**Comment by jckw** [comment]
Yep! Looks like you're right.

Fancy submitting a PR that also bumps [the version number](https://github.com/jckw/django-bootstrap-daterangepicker/blob/master/setup.py#L11) to 1.0.2?

**Comment by OmenApps** [comment]
Corrected with pull request #5
